app/src/features/apiClient/screens/apiClient/components/clientView/request/KeyValueForm.tsx

```diff
@@ -79,6 +80,17 @@ const KeyValueForm: React.FC<Props> = ({ keyValuePairs, setKeyValuePairs, keyOpt
                   defaultValue={param.key}
                   onChange={(val) => onKeyChange(val, index)}
                   variables={currentEnvironmentVariables}
+                  highlightConfig={{
+                    definedVariableClass: "highlight-defined-variable",
+                    undefinedVariableClass: "highlight-undefined-variable",
+                    pattern: /{{.*?}}/g,
+                    extractVariable: (matchedVariable: string) => matchedVariable.slice(2, -2),
+                  }}
+                  renderPopover={({ variable, position, variables }) => {
+                    return (
+                      <VariablePopover hoveredVariable={variable} popupPosition={position} variables={variables} />
```

Comment:
We can remove the `variables` param and can directly pass the`currentEnvironmentVariables` to variables prop